packages/loader/container-loader/src/index.ts

```diff
@@ -3,6 +3,6 @@
  * Licensed under the MIT License.
  */
 
+export * from "./connectionArgs";
```

Comment:
IConnectionArgs is internal thing, it should not be exposed from package. I do not see it used outside of this package, so why do we expose it?

Reply:
Based on this comment when IConnectionArgs was removed from exports. https://github.com/microsoft/FluidFramework/pull/7634#discussion_r719797111 

This was the comment generated after removing IConnectionArgs:
```
    // Warning: (ae-forgotten-export) The symbol "IConnectionArgs" needs to be exported by the entry point index.d.ts
    protected resumeInternal(args: IConnectionArgs): void;
```

Reply:
The `Container` class which is exported has the function `protected resumeInternal(args: IConnectionArgs): void` which relies on `IConnectionArgs`.

Reply:
Going to transform resumeInternal to private from protected to hide this